pages/index.tsx

```diff
@@ -225,6 +225,35 @@ export interface SuiteAndTest {
   ]
 }
 
+export interface Test {
+  duration: {
+    days: number
+    hours: number
+    minutes: number
+    months: number
+    seconds: number
+    years: number
+  }
+  test_history_id: number
+  test_id: number
+  name: string
+  test_history_resolution: number
```

Comment:
Juan, this `test_history_resolution` is a string in SuiteAndTest interface, so maybe change both, or just leave it as string for simplifying? Why number?

Reply:
I got this error if I keep it as a string, also I see that in [showResolution.js](https://github.com/delta-reporter/delta-frontend/blob/realtime-resolution-upt/components/templates/showResolution.js) is always used as a number as well, is returned as a string from the backend?

![Screenshot 2020-09-24 at 10 38 52](https://user-images.githubusercontent.com/14996980/94128515-2b924100-fe52-11ea-84d7-378a3bb8c47d.png)

Reply:
it's probably a number... I don't know, either compare it to `"1"` or change it to be a number everywhere :) 
Sorry, it's messy